Source/WebCore/dom/WindowEventLoop.h

```diff
@@ -63,8 +66,8 @@ class WindowEventLoop final : public EventLoop {
     WEBCORE_EXPORT static void breakToAllowRenderingUpdate();
 
 private:
-    static Ref<WindowEventLoop> create(const String&);
-    WindowEventLoop(const String&);
+    static Ref<WindowEventLoop> create(const Site&);
+    WindowEventLoop(const Site&);
```

Comment:
Perhaps it would be good to have an AgentClusterKey type, even if it just wraps Site for now, as it would make it match the spec language a bit better?